### PR TITLE
Fixes to ROS2RoboticManipulationTemplate

### PR DESCRIPTION
**Pull request by michalpelka**
Addresses #466

### CODE REVIEW

**Comment by adamdbrw** [comment]
Rebased (force pushed)

**Review by adamdbrw — APPROVED** [review]

File: Templates/Ros2RoboticManipulationTemplate/README.md [hunk 1]
```diff
@@ -4,7 +4,14 @@ This project is a template designed to help you quickly start simulating robotic
 
 This template focuses on robotic arm manipulation with ROS 2 and [MoveIt 2](https://moveit.picknik.ai/main/index.html).
 
+## Prerequisites 
 
+Make sure that your system follow [Requirements](https://development--o3deorg.netlify.app/docs/welcome-guide/requirements/).
```
Comment (adamdbrw) [inline Templates/Ros2RoboticManipulationTemplate/README.md:9]:
```suggestion
Make sure that your system meets [Requirements](https://development--o3deorg.netlify.app/docs/welcome-guide/requirements/).
```

File: Templates/Ros2RoboticManipulationTemplate/README.md [hunk 2]
```diff
@@ -4,7 +4,14 @@ This project is a template designed to help you quickly start simulating robotic
...
+
+Install necessary ROS2 packages required by this template:
+```bash
+sudo apt-get install ros-humble-moveit ros-humble-moveit-resources ros-humble-depth-image-proc
```
Comment (adamdbrw) [inline Templates/Ros2RoboticManipulationTemplate/README.md:13]:
```suggestion
sudo apt install ros-humble-moveit ros-humble-moveit-resources ros-humble-depth-image-proc
```

File: Templates/Ros2RoboticManipulationTemplate/README.md [hunk 3]
```diff
@@ -4,7 +4,14 @@ This project is a template designed to help you quickly start simulating robotic
 
 This template focuses on robotic arm manipulation with ROS 2 and [MoveIt 2](https://moveit.picknik.ai/main/index.html).
 
+## Prerequisites 
 
+Make sure that your system follow [Requirements](https://development--o3deorg.netlify.app/docs/welcome-guide/requirements/).
+
+Install necessary ROS2 packages required by this template:
```
Comment (adamdbrw) [inline Templates/Ros2RoboticManipulationTemplate/README.md:11]:
```suggestion
Install ROS2 dependencies for this template:
```

**Review by pijaro — APPROVED** [review]
Lgtm

File: Templates/Ros2RoboticManipulationTemplate/README.md [hunk 4]
```diff
@@ -4,7 +4,14 @@ This project is a template designed to help you quickly start simulating robotic
 
 This template focuses on robotic arm manipulation with ROS 2 and [MoveIt 2](https://moveit.picknik.ai/main/index.html).
 
+## Prerequisites 
 
+Make sure that your system meets [Requirements](https://development--o3deorg.netlify.app/docs/welcome-guide/requirements/).
+
+Install ROS2 dependencies for this template:
+```bash
+sudo apt install ros-humble-moveit ros-humble-moveit-resources ros-humble-depth-image-proc
```
Comment (jhanca-robotecai) [inline Templates/Ros2RoboticManipulationTemplate/README.md:13]: Is this template limited to ROS2 Humble or should we allow different versions as well?
```suggestion
sudo apt install ros-${ROS_DISTRO}-moveit ros-${ROS_DISTRO}-moveit-resources ros-${ROS_DISTRO}-depth-image-proc
```

**Review by jhanca-robotecai — APPROVED** [review]

**Pull request merged**